audited.gemspec

```diff
@@ -17,13 +17,14 @@ Gem::Specification.new do |gem|
 
   gem.required_ruby_version = '>= 2.3.0'
 
-  gem.add_dependency 'activerecord', '>= 4.2', '< 6.1'
+  gem.add_dependency 'activerecord', '>= 5.0', '< 6.1'
 
   gem.add_development_dependency 'appraisal'
-  gem.add_development_dependency 'rails', '>= 4.2', '< 6.1'
+  gem.add_development_dependency 'rails', '>= 5.0', '< 6.1'
   gem.add_development_dependency 'rubocop', '~> 0.54.0'
   gem.add_development_dependency 'rspec-rails', '~> 3.5'
   gem.add_development_dependency 'single_cov'
+  gem.add_development_dependency 'sprockets', '< 4.0'
```

Comment:
Sprockets >= 4 requires an app manifest file, and I'm not sure who to do that with the test rails app. If someone familiar with this repo could advise or update it, that would be appreciated.